backend/mora/cli/__init__.py

```diff
@@ -6,5 +6,29 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
 
+import os
+import sys
+
+# cant do monkeypatching only because file is used as program
```

Comment:
"cant" -> "Can't"

Reply:
> "cant" -> "Can't"

Alternatively, [cannot](https://en.oxforddictionaries.com/usage/cannot-or-can-not) 🙂